docs/_content/ontwikkelaars/algemeen/audit-trail.md

```diff
@@ -0,0 +1,248 @@
+API alle API's van componenten raakt. Echter, bij het NRC geeft de consumer 
+zelf aan waar deze API te vinden is. Bij de audit trail is hiervan geen sprake
+en dus moet de API consistent en snel te vinden zijn, en het is in essentie ook
+echt een onderdeel van het component.
+
+### Attributen
+
+`uuid` (automatisch)
+
+Unieke identificatie van de audit regel.
+
+`bron` (automatisch)
+
+De naam van get component waar de wijziging in is gedaan. Hoewel dit attribuut 
+in de context van de API niet relevant lijkt, aangezien de API onderdeel is van 
+het component. Mocht deze API echter gebruikt worden in een geaggregeerde 
+context, dan blijft het formaat voor consumers gelijk.
+
+`applicatieId` (verplicht)
+
+Unieke identificatie van de applicatie, binnen de organisatie. Deze komt 
+overeen met het `client_id` attribuut uit de JWT.
+
+`applicatieWeergave` (optioneel)
+
+Vriendelijke naam van de applicatie. Deze kan worden afgeleid uit het `label` 
+attribuut van de `Applicatie` resource in het Atorisatie Component (AC).
+
+`gebruikersId` (verplicht) 
+
+Unieke identificatie van de gebruiker, binnen de organisatie. Deze moet uit de 
+JWT worden gehaald.
+
+*TODO: Moet nog worden toegevoegd aan het JWT en relevante documentatie*
+
+`gebruikersWeergave` (optioneel)
+
+Vriendelijke naam van de gebruiker. Deze moet uit de JWT worden gehaald.
+
+*TODO: Moet nog worden toegevoegd aan het JWT en relevante documentatie*
+
+`actie` (verplicht)
+
+De uitgevoerde handeling *zoals bij notificaties*.
+
+`actieWeergave` (optioneel)
+
+Vriendelijke naam van de actie. Bijvoorbeeld als een relatie tussen 
+een document en een zaak wordt aangemaakt: "gekoppeld aan Zaak MOR-000001".
+
+*TODO: Wellicht ook toevoegen aan het NRC (buiten scope)*
+
+`resultaat` (verplicht)
+
+HTTP status code van de API response van de uitgevoerde handeling. Een HTTP
+status code van "200" of "201" is bijvoorbeeld een succesvol uitgevoerde 
+handeling, terwijl een "403" aangeeft dat de gebruiker of applicatie het wel
+geprobeerd heeft maar de handeling is niet uitgevoerd.
+
+`hoofdObject` (verplicht)
+
+De URL naar het hoofd object van een component *zoals bij notificaties*. Bij 
+het ZRC is dit dus altijd de URL naar een zaak.
+
+`resource` (verplicht)
+
+De resource naam *zoals bij notificaties*.
+
+`resourceUrl` (verplicht)
+
+De URL naar het object *zoals bij notificaties*.
+
+`resourceWeergave` (optioneel)
+
+Vriendelijke identificatie van het object. Bijvoorbeeld de zaak identificatie 
+of de omschrijving van een status.
+
+`toelichting` (optioneel)
+
+Toelichting waarom de handeling is uitgevoerd. Dit kan handig zijn voor sommige
+handelingen waarbij additionele context gewenst is, zoals het ophogen van een
+archiveringstermijn of een correctie buiten het reguliere proces om.
+
+`aanmaakdatum` (automatisch)
+
+De datum waarom de handeling is gedaan.
+
+#### TODO: Extra attributen
+
+In overweging om op te nemen:
+
+* `verschillen` -- Een lijst met de wijzigingen.
```

Comment:
Kunnen we de oude en nieuwe situatie teruggeven?

Reply:
Dat kan wel, in aanloop naar historie hadden we dat al uitgezocht. Ik zal em opnemen.

Reply:
Opgenomen als `wijzigingen`.